src/Core/src/Platform/Android/WrapperView.cs

```diff
@@ -55,6 +55,7 @@ protected override void OnLayout(bool changed, int left, int top, int right, int
 			var widthMeasureSpec = MeasureSpecMode.Exactly.MakeMeasureSpec(right - left);
 			var heightMeasureSpec = MeasureSpecMode.Exactly.MakeMeasureSpec(bottom - top);
 
+			_invalidateShadow = true;
```

Comment:
Do we need to do this is the positions (X/Y) changes as well, or just size? I can't recall if the shadow is automatically positioned based on the view origin or also gets "stuck"

Reply:
We can do this if it actually becomes a problem.